Can't run tests for the package geiger

I was trying to understand the project test suite and stumbled upon the fact that both `cargo -p geiger test` and `cargo test` in the `geiger` directory fail for me with the following error:

```
kirill@kdesktop:~/ws/rust/cargo-geiger$ cargo test -p geiger 
   Compiling cargo-geiger-serde v0.1.0 (/home/kirill/ws/rust/cargo-geiger/cargo-geiger-serde)
error[E0277]: the trait bound `semver::Version: package_id::_::_serde::Deserialize<'_>` is not satisfied
  --> cargo-geiger-serde/src/package_id.rs:12:5
   |
12 |     /// Package version
   |     ^^^^^^^^^^^^^^^^^^^ the trait `package_id::_::_serde::Deserialize<'_>` is not implemented for `semver::Version`
   |
   = note: required by `package_id::_::_serde::de::SeqAccess::next_element`

error[E0277]: the trait bound `semver::Version: package_id::_::_serde::Deserialize<'_>` is not satisfied
  --> cargo-geiger-serde/src/package_id.rs:12:5
   |
12 |     /// Package version
   |     ^^^^^^^^^^^^^^^^^^^ the trait `package_id::_::_serde::Deserialize<'_>` is not implemented for `semver::Version`
   |
   = note: required by `package_id::_::_serde::de::MapAccess::next_value`

error[E0277]: the trait bound `semver::Version: package_id::_::_serde::Serialize` is not satisfied
  --> cargo-geiger-serde/src/package_id.rs:12:5
   |
12 |     /// Package version
   |     ^^^^^^^^^^^^^^^^^^^ the trait `package_id::_::_serde::Serialize` is not implemented for `semver::Version`
   |
   = note: required by `package_id::_::_serde::ser::SerializeStruct::serialize_field`

error: aborting due to 3 previous errors

For more information about this error, try `rustc --explain E0277`.
error: could not compile `cargo-geiger-serde`.

To learn more, run the command again with --verbose.
```

--- a/cargo-geiger-serde/Cargo.toml
+++ b/cargo-geiger-serde/Cargo.toml
@@ -10,6 +10,6 @@ repository = "https://github.com/rust-secure-code/cargo-geiger"
 version = "0.1.0"
 
 [dependencies]
-semver = "0.11.0"
+semver = { version = "0.11.0", features = ["serde"] }
 serde = { version = "1.0.116", features = ["derive"] }
 url = { version = "2.1.1", features = ["serde"] }

